app/src/main/assets/lang/en_US.json

```
@@ -1,103 +1,103 @@
 {
-    "category": {
-        "spying_privacy": "Spying & Privacy",
-        "media_manager": "Media Manager",
-        "ui_tweaks": "UI & Tweaks",
+    "Kategorie": {
+        "spying_privacy": "Spähen & Privacy",
```

This sounds a bit funny in german. I would rather write
```suggestion
        "spying_privacy": "Spy & Privatsphäre",
```
But i am also not sure if we should just leave this one be.
Not everything should be translated

---

Yeah english would sound better here, true.